tools/theme/test/themes.test.ts

```diff
@@ -21,11 +21,11 @@ type TestableThemeConstructor = {
     themeFragmentsByKind: ThemeFragmentMap;
 };
 
-describe('Themes', () => {
+describe('Systems', () => {
```

Comment:
We should include one or two tests for the deprecated `theme` attribute, specifically the failure notes above where an end user cannot set `theme` more than once right now.

Reply:
Tests has been added